### PR TITLE
new-pr-lambda-translate

### PR DESCRIPTION
**Pull request by abilashkumar**
*Issue #, if available:* NA

*Description of changes:*

This is a new pattern that is being requested to include which performs translation
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lambda-translate/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd _patterns-model
```
Comment (mavi888) [inline lambda-translate/README.md:24]: update with the right path

File: lambda-translate/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+1. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## How it works
+
+Use the AWS CLI to invoke the Lambda function.
+
+The example provided  performs Translation from English to German language. The following lanugaes are [supported](https://docs.aws.amazon.com/translate/latest/dg/what-is-languages.html).
```
Comment (mavi888) [inline lambda-translate/README.md:43]: lanugaes - Typo :)

File: lambda-translate/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Testing
+
+Replace "{LambdaFunctionName}" with the function name as seen in the output of CloudFormation template
+
+```
+aws lambda invoke --function-name lambda-sam-1-TranslateTextLambdaFunction-N34oQq55jama --invocation-type RequestResponse --cli-binary-format raw-in-base64-out --payload "{\"text\":\"I am very happy\", \"sl\":\"en\",\"tl\":\"de\"}" response.json
```
Comment (mavi888) [inline lambda-translate/README.md:50]: in the test you say to replace the lambda function name with that and then in the example you put the full name. fix it for consistency

File: lambda-translate/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+The command above returns the following output:
```
Comment (mavi888) [inline lambda-translate/README.md:53]: the response is in the response.json not in the command for this particular command. Fix

File: lambda-translate/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+```
+das ist schlecht
+```
+
+## Cleanup
+ 
+1. Delete the stack
+    ```bash
+    aws cloudformation delete-stack --stack-name STACK_NAME
```
Comment (mavi888) [inline lambda-translate/README.md:62]: use sam delete command instead

File: lambda-translate/example-pattern.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    ]
+  },
+  "testing": {
+    "text": [
+      "See the Github repo for detailed testing instructions."
+    ]
+  },
+  "cleanup": {
+    "text": [
+      "Delete the stack: aws cloudformation delete-stack --stack-name {STACK_NAME}"
```
Comment (mavi888) [inline lambda-translate/example-pattern.json:46]: use sam delete

File: lambda-translate/example-pattern.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    ]
+  },
+  "authors": [
+    {
+      "name": "Abilashkumar P C",
+      "image": "https://phonetool.amazon.com/users/abilasc",
```
Comment (mavi888) [inline lambda-translate/example-pattern.json:52]: please send a public link for image

File: lambda-translate/README.md (hunk 1 above)
Comment (abilashkumar) [reply lambda-translate/README.md]: Updated the path

File: lambda-translate/README.md (hunk 2 above)
Comment (abilashkumar) [reply lambda-translate/README.md]: Modified

File: lambda-translate/README.md (hunk 3 above)
Comment (abilashkumar) [reply lambda-translate/README.md]: Fixed the above error

File: lambda-translate/README.md (hunk 4 above)
Comment (abilashkumar) [reply lambda-translate/README.md:53]: Updated

File: lambda-translate/example-pattern.json (hunk 6 above)
Comment (abilashkumar) [reply lambda-translate/example-pattern.json]: Added `sam delete`

File: lambda-translate/example-pattern.json (hunk 7 above)
Comment (abilashkumar) [reply lambda-translate/example-pattern.json]: Updated

File: lambda-translate/README.md (hunk 5 above)
Comment (abilashkumar) [reply lambda-translate/README.md]: Modified

File: lambda-translate/README.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd patterns-model
```
Comment (mavi888) [inline lambda-translate/README.md:24]: cd lambda-translate need to say

File: lambda-translate/README.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+Use the AWS CLI to invoke the Lambda function.
+
+The example provided  performs Translation from English to German language. The following languages are [supported](https://docs.aws.amazon.com/translate/latest/dg/what-is-languages.html).
+
+## Testing
+
+Replace "{LambdaFunctionName}" with the function name as seen in the output of CloudFormation template
+
+```
+aws lambda invoke --{LambdaFunctionName} lambda-sam-1-TranslateTextLambdaFunction-N34oQq55jama --invocation-type RequestResponse --cli-binary-format raw-in-base64-out --payload "{\"text\":\"I am very happy\", \"sl\":\"en\",\"tl\":\"de\"}"
```
Comment (mavi888) [inline lambda-translate/README.md:50]: this command doesnt work. please fix it

File: lambda-translate/README.md (hunk 4 above)
Comment (mavi888) [reply lambda-translate/README.md:53]: this was not fixed

File: lambda-translate/README.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+The command above returns the following output:
+```
+das ist schlecht
+```
+
+## Cleanup
+ 
+Delete the stack - Replace "{StackName}" with the name of the stack
```
Comment (mavi888) [inline lambda-translate/README.md:60]: the format for this line is wrong

**Comment by mavi888** [comment]
This PR still has pending issues. Please test the PR and the instructions before submiting it.

**Comment by mavi888** [comment]
@abilashkumar if you don't reply to this pr in the next 7 days I will close it.
Cheers

**Comment by mavi888** [comment]
Due to lack of activity in this PR I will close it.
@abilashkumar if you fix the comments please feel free to reopen the PR and we can work from there.

Cheers,
Marcia

**Pull request closed without merging**